### PR TITLE
tweepy.client script

### PR DESCRIPTION
**Pull request by ashioyajotham**
First attempt at scraping twitter.

### CODE REVIEW

File: 1. Data Collection/twitter.py
Comment (VictorOmondi1997) [inline 1. Data Collection/twitter.py:1]: This looks great in helping us get started with this project. Thank you for putting it in place

**Review by VictorOmondi1997 — APPROVED** [review]
This looks great

File: 1. Data Collection/twitter.py
Comment (ashioyajotham) [reply 1. Data Collection/twitter.py:1]: @Stephen-Kamau let me look into that

Comment (3liud) [reply 1. Data Collection/twitter.py:1]: Thanks, @ashioyajotham, for this script, and thanks @Stephen-Kamau, for the suggestions. Once that suggestion is implemented, we can move on to the next step.

**Pull request merged**